config/kube-app-manager/manager.yaml

```diff
@@ -1,17 +1,10 @@
 # Copyright 2020 The Kubernetes Authors.
 # SPDX-License-Identifier: Apache-2.0
-
-apiVersion: v1
-kind: Namespace
-metadata:
-  labels:
-    controller-tools.k8s.io: "1.0"
-  name: system
 ---
 apiVersion: v1
 kind: Service
 metadata:
-  name: controller-manager-service
+  name: service
```

Comment:
the name is too generic.

Reply:
it uses kustomize namePrefix
it's replaced with `$prefix-service`
as shown in:
https://github.com/kubernetes-sigs/application/pull/179/files#diff-d0ae248f731b5b6f88b3f9e81d035754R718

Reply:
ack